plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetWriterConfig.java

```diff
@@ -72,6 +73,20 @@ public ParquetWriterOptions toParquetWriterOptions()
         return ParquetWriterOptions.builder()
                 .setMaxBlockSize(getBlockSize())
                 .setMaxPageSize(getPageSize())
+                .setRowGroupMaxRowCount(getRowGroupMaxRowCount())
```

Comment:
pre-existing so optional, but more future-proof would be to keep Options as a field here, like in `io.trino.plugin.hive.parquet.ParquetReaderConfig#options`.

Reply:
I was thinking that too, but it doesn't seem to work well with session property overrides. The `Iceberg/Hive SessionProperties` class needs a non-null default value and I don't think it supports optional properties so we can't just pass Optional.empty the whole way through if the value isn't set.

Reply:
i don't understand the explanation. nvrm, let's talk this separately / in the future